spec/send_spec.js

```diff
@@ -114,8 +106,6 @@ describe('send', function() {
         monitoring.showHideList();
 
         authoring.sendTo('Politic Desk');
-        //Spell check confirmation modal save action
-        authoring.confirmSendTo();
```

Comment:
it should stay there, problem is the implementation not working with promises..
it's fixed here https://github.com/superdesk/superdesk-client-core/pull/91/files#diff-5c9045fd902f440d5b002c381e24165cR77